<launch>
<node pkg="forklift_server" type="Topology_map_server.py" name="TopologyMap_server" output="screen">
    <param name="odom" value="/wheel_odom" /><!--里程計 Topic（map -> base_link, 是tf相對關係, 非輪式里程計）/-->
    <param name="start_node" value="P1" />
    <rosparam param= "graph">
    {
        "P1": {"P2": 1},
        "P2": {"P3": 1},
        "P3": {"P4": 1},
        "P4": {"P5": 1},
        "P5": {"P5": 1}
    }
    </rosparam>
    <rosparam param= "waypoints">
    {    
        "P1": [-6.605, 1.376, -0.675, 0.738],
        "P2": [-6.458, -3.399, -0.708, 0.707],
        "P3": [-6.544, -6.050, -0.716, 0.716],
        "P4": [-6.458, -3.399, -0.708, -0.707],
        "P5": [-6.544, -6.050, -0.716, 0.716]
    }
    </rosparam>
</node>
</launch>

        <!-- "P0": [-6.605, 1.376, -0.675, 0.738],        
        1
        "P1_E1": [-6.458, -3.399, -0.708, 0.707],    
        2 4  10
        "P1_W1": [-6.458, -3.399, -0.708, -0.707],   
        8
        "P1":    [-6.458, -3.399, -0.013, 1.000],    
        3 9 

        "P2_E1": [-6.544, -6.050, -0.716, 0.716],    
        5
        "P2":    [-6.544, -6.050, -0.013, 1.000],    
        6
        "P2_W1": [-6.544, -6.050, -0.716, -0.716],   
        7

        "P3":    [-6.504, -8.328, -0.699, 0.715],    
        11
        "P5":    [-6.529, -8.340, -0.006, 1.000] -->
